storm-core/test/clj/backtype/storm/supervisor_test.clj

```diff
@@ -261,8 +260,8 @@
                                 (str "-Dstorm.id=" mock-storm-id)
                                 (str "-Dworker.id=" mock-worker-id)
                                 (str "-Dworker.port=" mock-port)
-                               "-Dstorm.log.dir=/logs"
-                               "-Dlog4j.configurationFile=/log4j2/worker.xml"
+                                (str "-Dstorm.log.dir=" file-path-separator "logs")
+                                (str "-Dlog4j.configurationFile=" file-path-separator "log4j2" file-path-separator "worker.xml")
```

Comment:
@hustfxj 
STORM-1463 is merged into master (2.x) and 1.x, and this pull request is for 0.10.x.
While this PR can still make unit tests succeed, we may want to backport STORM-1463 to 0.10.x-branch and reflect change to supervisor_test.clj.